int32 roboyID
float64 F_contact
float64 d_lift
float64 d_prep
float64 F_max
float64 psi_heading
float64 omega_heading
float64 v_forward
float64 v_COM
float64 k_v
float64 k_h
float64[] k_p_theta_left
float64[] k_p_theta_right
float64[] k_d_theta_left
float64[] k_d_theta_right
float64[] k_p_phi
float64[] k_d_phi
float64 k_V
float64 k_P
float64 k_Q
float64 k_omega
float64 k_M_Fplus
float64 c_hip_lift
float64 c_knee_lift
float64 c_stance_lift
float64 c_swing_prep
float64[] theta_groin_0
float64[] phi_groin_0
float64 theta_trunk_0
float64 phi_trunk_0
float64[] theta_knee
float64[] theta_ankle
float64[] d_s
float64[] d_c
float64[] v_s
float64[] v_c
float32 sim_time